tests/unit/assessment/test_dashboard.py

```diff
@@ -33,6 +33,8 @@ def download_mock(path):
                     {
                         "jobs.sql:query_id": "91e51760-7653-4769-bc32-1595ce1892af",
                         "all_tables.sql:query_id": "4000e54c-4c51-45b3-b009-a4dd9a3b5599",
+                        "assessment_azure_05_0_azure_service_principals.sql:query_id"
+                        "": "11e51760-7653-4769-bc32-1595ce1892af",
```

Comment:
this looks like a bug. can you investigate why?

Reply:
corrected